examples/mutate/abstract functionality blueprint/pipe_output_on_output.py

```diff
@@ -3,15 +3,15 @@
 from hamilton.function_modifiers import extract_fields, pipe_output, step
 
 
-def _pre_step(something: int) -> int:
+def pre_step(something: int) -> int:
     return something + 10
 
 
-def _post_step(something: int) -> int:
+def post_step(something: int) -> int:
     return something + 100
 
 
-def _something_else(something: int) -> int:
+def something_else(something: int) -> int:
     return something + 1000
```

Comment:
so my concern here is that people can now mix functions in a module that aren't hamilton nodes; i.e. you don't know what you're looking at...

Reply:
@skrawcz yep, agreed, see this comment (based on yours): https://github.com/DAGWorks-Inc/hamilton/pull/1172#discussion_r1792881453

Reply:
so my thought is that we don't enable `@ignore` here.